src/main/java/seedu/anilist/model/ModelManager.java

```diff
@@ -42,6 +42,7 @@ public ModelManager(ReadOnlyAnimeList animeList, ReadOnlyUserPrefs userPrefs) {
         this.filteredAnime = new FilteredList<>(this.animeList.getAnimeList());
         this.currentTab = new TabOption("all");
         this.tabOptionFilter = PREDICATE_SHOW_ALL_ANIME;
+        updateUserStats();
```

Comment:
Why is this suddenly needed?

Reply:
I realised while writing tests that Stats is the only field that's uninitialised, so I just initialised it here. makes testing easier too